Ark.Tools.ResourceWatcher/ResourceWatcher.cs

```diff
@@ -145,7 +145,7 @@ protected virtual async Task _runOnce(RunType runType, CancellationToken ctk = d
 
                 if (_config.SkipResourcesOlderThanDays.HasValue)
                     infos = infos
-                            .Where(x => _getModified(x).Value.Date > LocalDateTime.FromDateTime(now).Date.PlusDays(-(int)_config.SkipResourcesOlderThanDays.Value))
+                            .Where(x => _getEarliestModified(x).Value.Date > LocalDateTime.FromDateTime(now).Date.PlusDays(-(int)_config.SkipResourcesOlderThanDays.Value))

```

Comment:
Since earliest is guaranteed not to be NULL, please return a LocalDateTime (not nullable).